nucleus/tests/quicklook/pom.xml

```diff
@@ -41,38 +41,53 @@ tests.  The file will automatically be picked up as part of the suite. There is
 no need to modify this pom.xml file or any other configuration file. Useful
 utilities are in the NucleusTestUtils class.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.glassfish.main.tests</groupId>
         <artifactId>nucleus-tests</artifactId>
         <version>6.0.0-SNAPSHOT</version>
     </parent>
-    <groupId>org.glassfish.main.tests</groupId>
+
     <artifactId>nucleus-quicklook</artifactId>
+
     <name>Nucleus Quicklook Tests</name>
-    <version>6.0.0-SNAPSHOT</version>
     <description>This pom describes how to run QuickLook tests on the Nucleus distribution</description>
+
     <dependencies>
         <dependency>
             <groupId>org.glassfish.main.common</groupId>
             <artifactId>common-util</artifactId>
             <version>${project.version}</version>
             <scope>test</scope>
-            <type>jar</type>
         </dependency>
         <dependency>
             <groupId>org.glassfish.main.tests</groupId>
             <artifactId>utils-ng</artifactId>
             <version>${project.version}</version>
             <scope>test</scope>
-            <type>jar</type>
         </dependency>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M4</version>
+                <configuration>
+                    <systemPropertyVariables>
+                        <nucleaus.home>${nucleaus.home}</nucleaus.home>
```

Comment:
Is that nucle**a**us on purpose or typo?

Reply:
I hope it's a typo!  @arjantijms ?